core/src/main/scala/kafka/metrics/KafkaMetricsConfig.scala

```diff
@@ -20,18 +20,20 @@
 
 package kafka.metrics
 
-import kafka.utils.{VerifiableProperties, CoreUtils}
+import kafka.server.{Defaults, KafkaConfig}
+import kafka.utils.{CoreUtils, VerifiableProperties}
 
 class KafkaMetricsConfig(props: VerifiableProperties) {
 
   /**
    * Comma-separated list of reporter types. These classes should be on the
    * classpath and will be instantiated at run-time.
    */
-  val reporters = CoreUtils.parseCsvList(props.getString("kafka.metrics.reporters", ""))
+  val reporters = CoreUtils.parseCsvList(props.getString(KafkaConfig.KafkaMetricsReporterClassesProp, ""))
```

Comment:
Thanks for the update. Just one more minor comment. Would it be slightly better to also define `KafkaMetricsReporterClasses = ""` in `kafka.serverDefaults`, similar to the existing `Defaults.MetricReporterClasses = ""`? It seems that in the existing KafkaConfig.java, we always use use a default variable with value `""` rather than using `""` directly. Otherwise LGTM.

Reply:
@lindong28  Thanks for the review. Updated the PR.